### ISSUE TITLE
feedback

### ISSUE THREAD

**Issue by realfinder** [SAPikachu/MP_Pipeline#1]
hi, It seems you didn't log in in doom9 since 2015

I post this there http://forum.doom9.org/showpost.php?p=1767663&postcount=236

1- I wonder if you can support

ClearAutoloadDirs()
AddAutoloadDir("path")

it's one of Avisynth+ features http://forum.doom9.org/showthread.php?p=1646315&highlight=directories#post1646315

they can't be used inside MPP because MPP load it's plugin in every block, so I think we need something like

```
### top start ###
some lines here
### top end ###
```

edit: this better
```
ClearAutoloadDirs() #topmpp
AddAutoloadDir("path") #topmpp
```

make the mpp looking for lines that has #topmpp and put it in the top of all things

2- I wonder if you can make MPP reload the blocks that crash in the middle of encoding (With an option to disable it for testing purposes or something, MP_Pipeline("blocks", reload=false) something like that)

thanks

**Comment by SAPikachu** [comment]
Thanks for the suggestion, but unfortunately I won't have much time to implement them. Patches welcome. :)

**Comment by realfinder** [comment]
take your time, For me I can't deal with C++, I did support setmtmode last time in a distorted manner

and I feel that my compile not stable, it's tend to random crash

**Comment by realfinder** [comment]
btw, what "Always kill children" do?

**Comment by SAPikachu** [comment]
It specifies whether to kill slave processes when cleaning up, instead of letting them exit cleanly. This helps if there is bug in filters used in slave processes that makes the process freeze on exit, but filters like tfm may not be able to write stats file if this option is specified.

**Comment by realfinder** [comment]
hi SAPikachu, any news?

**Comment by SAPikachu** [comment]
Well, as I mentioned in my earlier comment, I really have no time for this anymore. Sorry. If anyone can contribute a patch, I will be happy to merge it.

**Comment by realfinder** [comment]
let see if @msg7086 can do it :)

and I think audio support is important too, also it's used in mvtools2, see here http://forum.doom9.org/showthread.php?p=1671402#post1671402

**Comment by msg7086** [comment]
@realfinder Not really. I'm not a C++ user myself, and what I've done is just a small and quick fix to the existing feature that I personally would use frequently. Sorry to let you down.

**Comment by FranceBB** [comment]
I would really like to have audio support as well, 'cause it would be really helpful.

**Comment by msg7086** [comment]
@FranceBB Please be advised that no active development -- apart from some crash fixes -- occurs at the moment.
